packages/site/.eleventy.js

```diff
@@ -4,7 +4,8 @@ module.exports = eleventyConfig => {
         'landing/dist': './',
         '../../packages/nimble-components/dist/storybook': 'storybook',
         '../../angular-workspace/dist/example-client-app': 'storybook/example-client-app',
-        '../../packages/nimble-blazor/dist/blazor-client-app': 'storybook/blazor-client-app'
+        '../../packages/nimble-blazor/dist/blazor-client-app': 'storybook/blazor-client-app',
+        '../../packages/performance/dist': 'storybook/performance'
```

Comment:
Should need to remap each component test that is added here, made the performance package build ouput like:

dist/component-name/*

so we can just copy all the contents in one copy